test/static/signatures/valid/response.root-signed.assertion-signed.xml

```diff
@@ -1,16 +1,24 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<samlp:Response xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion" Destination="https://evil-corp.madness.com/sso/callback" ID="_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" InResponseTo="_e8df3fe5f04237d25670" IssueInstant="2015-08-31T08:54:06+00:00" Version="2.0">
-    <saml:Issuer>https://evil-corp.com</saml:Issuer>
+<?xml version="1.0"?>
```

Comment:
I'm not sure, but is specifying the encoding of the XML file required, good practice, etc? Or is UTF8 the default?

Reply:
this was not intentional, i'll add the encoding back in